src/main/relation/query_relation.cpp

```diff
@@ -62,6 +50,42 @@ unique_ptr<TableRef> QueryRelation::GetTableRef() {
 	return std::move(subquery_ref);
 }
 
+BoundStatement QueryRelation::Bind(Binder &binder) {
+	auto saved_binding_mode = binder.GetBindingMode();
+	binder.SetBindingMode(BindingMode::EXTRACT_REPLACEMENT_SCANS);
+	bool first_bind = columns.empty();
+	auto result = Relation::Bind(binder);
+	auto &replacements = binder.GetReplacementScans();
```

Comment:
Does this now have different semantics:

```py
con.sql('SELECT * FROM df').create_view('v1)
```

From this:

```py
con.execute('CREATE VIEW v1 AS SELECT * FROM df')
```

Might not be too much of a problem but something to be aware of. We might want to unify all of this in the future.

Reply:
Added a test and a comment on the currently expected error that we should unify the behavior 👍